README.md

```diff
@@ -8,8 +8,8 @@ Odoo Connector Modules
 
 This OpenERP add-on has a modular and generic core, with the ability to be extended with additional modules for new features or customizations.
 
-The team for the community is here: https://launchpad.net/~openerp-connector-community
-Please join and subscribe to the mailing list.
+Feel free to subscribe on the mailing list:
```

Comment:
Well, all lists are available in that link, but we should guide people to which ML is the adequate for this, because you know that sometimes users can be very "confused"